ui/test/cypress/plugins/unit.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-env node */
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+import { injectDevServer } from '@quasar/quasar-app-extension-testing-e2e-cypress/cct-dev-server'
+
+const {
+  addMatchImageSnapshotPlugin
+} = require('cypress-image-snapshot/plugin')
```

Comment:
Can't this be imported using `ES` import syntax as we do for `injectDevServer`?

Reply:
Yes it can! Will change